src/OpenTelemetry.Api/Trace/SpanAttributeConstants.cs

```diff
@@ -40,9 +40,10 @@ public static class SpanAttributeConstants
         public const string HttpRouteKey = "http.route";
         public const string HttpFlavorKey = "http.flavor";
 
-        public const string DatabaseTypeKey = "db.type";
-        public const string DatabaseInstanceKey = "db.instance";
+        public const string DatabaseSystemKey = "db.system";
```

Comment:
can you add this if feasible?
https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/database.md#connection-level-attributes-for-specific-technologies

Reply:
Created #782 for this.